### ISSUE TITLE
If this project is abandoned, please refer to any alternative in README, 

### ISSUE THREAD

**Issue by q2dg** [scantailor/scantailor#303]
Thanks

**Comment by ncraun** [comment]
Any suggestions?

**Comment by barrer** [comment]
It still works very well now!   
The new operating system may not be very compatible, but you can try running it with a virtual machine.   
The image format has not changed much for many years, so it is still very good!

**Comment by barrer** [comment]
You can try this author's scanned image processing software:  
https://www.cnblogs.com/stronghorse/p/4913182.html  
download the "ComicEnhancer Pro" software

Part of the tutorial:  
Https://www.cnblogs.com/stronghorse/p/9425046.html

Use Google Translate to check it out.

**Comment by Tiesenhausen** [comment]
This is an excellent program that should be on the computer of everyone who
works with pdf documents, particularly scanned pages. Produce a version
that is end-user friendly, with documentation to explain the functions,
post it online for downloading, and send copies to various writers and
websites that will publicize it. There is nothing else like it. Once it
becomes known, momentum will build up, a professional edition will become
viable, and the resources for updates and improvements will become
available.

I use it constantly and would recomment it to anybody--for example, Google
Books with their ugly pdfs.

Michael L. Bates

**Comment by hanklank** [comment]
Love this program, it has a lot of potential, and I hope someone gets the time to polish and update it.

**Comment by Tiesenhausen** [comment]
I agree, and believe this program could be commercially viable if someone
would invest the time 2 make it more user-friendly and to promote it.
Michael Bates

**Comment by Piolie** [comment]
There have been a few forks of this project since it was last updated. I prefer [ST Advanced](https://github.com/4lex4/scantailor-advanced), but there is also [ST Universal](https://github.com/trufanov-nok/scantailor) as well as some (mostly stalled) others (ST Duo, ST Enhanced, ST Experimental, ST Featured, ST Plus). Most features of those have been integrated in the former.
